com.unity.render-pipelines.universal/Documentation~/configuring-universalrp-for-use.md

```diff
@@ -4,6 +4,7 @@ To configure and use the Universal Render Pipeline (URP), you must first:
 
 - create the Universal Render Pipeline Asset
 - add the Asset to the Graphics settings for your Project
+- ensure that no other Scriptable Render Pipeline Asset is assigned in any of the quality levels
```

Comment:
Isn't it the opposite, we should make sure that we have a SRP asset in each quality settings, there's actually plan to remove the graphics settings one. Or do you mean to ensure there's no other SRP asset from a different pipeline (HDRP) in the quality settings levels?

Reply:
I didn't know there was a plan to remove the graphics settings one. 

My thinking was the if you create a pipeline asset and assign it in graphics settings, you can't be certain that it will actually be used (because it might be overridden by quality settings).
Maybe I rephrase to create one per quality level and assign them there?

Reply:
I rephrased to be more explicit about quality levels